lib/main-app.js

```diff
@@ -3,6 +3,7 @@ const app = electron.app
 const Menu = electron.Menu
 const ipc = electron.ipcMain
 const GhReleases = require('electron-gh-releases')
+const isDev = require('electron-is-dev') // isDev is True if running in development mode. Overide by setting ELECTRON_IS_DEV environment variable to 1
```

Comment:
Do we really need a package just for checking the app is in development? Can we just use the old fashion way like this?
```
const isDev = process.env.NODE_ENV !== 'production'
```

Reply:
The merits between those two choices I'm not sure of, this is just the recommended method from the official Electron guides.

Reply:
electron-is-dev is pretty lightweight 
![image](https://user-images.githubusercontent.com/21070668/51081054-9827d500-169b-11e9-95fd-e67dc7cdb20d.png)

Reply:
I think the old way is enough for this task because `electron-is-dev` does almost the same thing:
![image](https://user-images.githubusercontent.com/12984316/51086193-0214ad00-1776-11e9-93ff-a5aa76129a16.png)